en/installconfig-lxd-install.md

```diff
@@ -4,16 +4,25 @@ TODO:  Text needs a review
 
 # Install with LXD
 
+[LXD][link-lxd] is a next generation system container manager for Linux
+containers, offering a user experience similar to virtual machines without the
+same overheads.
+
 Installing MAAS into containers is a good choice for users who want to test
 MAAS, or who may want to continue leveraging an existing container architecture
 or policy. 
 
 MAAS running with LXD has the following requirements:
 
-- a network bridge on the LXD host (e.g. br0)
+- a network bridge on the LXD host (e.g. lxdbr0)
 - LXD and ZFS
 - a container profile
 
+!!! Note:
+    The LXD project was founded and is currently led by
+    [Canonical Ltd][link-canonical] with contributions from a range of other companies 
+    and individual contributors.
+
```

Comment:
I don't see why we need to include this in the MAAS docs.

Reply:
Just to give some context. This and the para above is really if anyone is browsing the install docs to see what's possible and may not be familiar with LXD.